util/check-format.pl

```diff
@@ -13,8 +13,9 @@
 #
 # usage:
 #   check-format.pl [-l|--sloppy-len] [-l|--sloppy-bodylen]
-#                   [-s|--sloppy-spc] [-c|--sloppy-cmt] [-m|--sloppy-macro]
-#                   [-h|--sloppy-hang] [-1|--1-stmt]
+#                   [-s|--sloppy-spc] [-c|--sloppy-cmt]
```

Comment:
It's `--sloppy-space` in the script (now).

To be honest, I'd prefer long names, so `--sloppy-comment` too.

Reply:
> It's `--sloppy-space` in the script (now).

Oops, made consistent again.

> To be honest, I'd prefer long names, so `--sloppy-comment` too.

Yes, this is more readable. Changed as well.